### DISCUSSION TITLE
Significant number of CVEs detected in version 0.55, but none in version 0.55.1

### DISCUSSION THREAD

**Discussion by lerouxr** [aquasecurity/trivy#7503] [Q&A] [answered]
### Question

We have recently observed a considerable increase in CVEs (around 52) when running our CI scans with version 0.55 for a spring sproject (3.2.5). However, after upgrading to version 0.55.1, all CVEs were resolved, with none being reported.

This sudden change between versions raises questions about what exactly happened during the transition from 0.55 to 0.55.1.

CI Scan Results:
Version 0.49.1: No CVEs detected.
Version 0.54.1: No CVEs detected.
Version 0.55: ~52 CVEs detected.
Version 0.55.1: No CVEs detected.
We would like to understand the following:

What caused the spike in CVEs with version 0.55?
What changes in version 0.55.1 resolved all CVEs?
Is there a potential misconfiguration in our CI pipeline, or were these legitimate security issues fixed between versions?
Any insights from the development team or maintainers regarding these findings would be highly appreciated.

Thank you!

### Target

Filesystem

### Scanner

Vulnerability

### Output Format

None

### Mode

Standalone

### Operating System

Alpine Linux 

### Version

```bash
Version: 0.55.0
```

**Comment by knqyf263** [comment, marked as answer]
We had a bug in v0.55.0 and reverted in v0.55.1.
https://github.com/aquasecurity/trivy/pull/7488

**Comment by lerouxr** [comment]
Thank you very much for your prompt and clear response. I truly appreciate it.
